ui/app/src/views/core/Footer.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Theme } from '@mui/material';
+import { SxProps } from '@mui/system/styleFunctionSx/styleFunctionSx';
+import { Github } from 'mdi-material-ui';
+
+const style: SxProps<Theme> = {
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'center',
+  alignItems: 'center',
+  '& li': {
+    listStyle: 'none',
+    display: 'inline',
+    fontSize: '0.8rem',
+  },
+  '& li + li:before': {
+    content: '"|"',
+    padding: '3px',
+  },
+  '& a:visited': {
+    color: 'inherit',
+  },
+};
+
+export function FooterView(): JSX.Element {
```

Comment:
Wouldn't you want to make this the default export, since there's only one thing? Same for the header.

Apropos this signature (with explicit return value), I'm more used to React components being typed and written like:

```
import React, { FC } from 'react';

export default Footer: FC = () => <Box>...</Box>
```

...then the return value is inferred from that, and you can also include typing for props like `FC<FooProps>` when needed.

But seems like besides the typing, for components we so far use the `function` keyword rather than arrow functions in Perses (in Prometheus we do it the other way around)?

Reply:
Just my $0.02 but I think either syntax is fine for components (`function Blah() {}` or `const Blah = () => {}`). Most React codebases I've been in have tended to mix and match. Personally I gravitate towards `function`, but it's totally just a style thing I probably picked up years ago, maybe before we had arrow functions. 🤷‍♂️ 

RE: Default exports though, that's actually a convention that I'm pretty sure React devs picked up because if you ever go to do code-splitting, [`React.lazy`](https://reactjs.org/docs/code-splitting.html#reactlazy) for lazy-loading components only works with components that are default exports out of the box, so I think we all got in the habit of just always making components be default exports.